test/test_jit.py

```diff
@@ -12803,6 +12803,35 @@ def test({arg_str}):
 
             FileCheck().check_not("prim::PythonOp").run(cu.test.graph)
 
+    def test_isinstance_metacompile(self):
```

Comment:
This should also check something where the value could be either type, i.e. a class attribute

```python
class M(torch.nn.Module):
	def __init__(self, x):
		self.x = x
	...

torch.jit.script(M(2))
torch.jit.script(M('hi'))
```

Reply:
`isinstance` is supposed to throw if it can't be statically resolved (don't have runtime support); that's an existing [limitation](https://github.com/pytorch/pytorch/blob/8fc349f7be84de74fbdadaca518a69af947d6739/torch/csrc/jit/script/compiler.cpp#L2181)

Reply:
It is still statically resolved here, each module statically has a different type for that attribute name